streams/src/main/java/org/apache/kafka/streams/processor/internals/assignment/ClientState.java

```diff
@@ -52,15 +52,15 @@ public ClientState() {
     }
 
     ClientState(final int capacity) {
-        this(new HashSet<>(),
-             new HashSet<>(),
-             new HashSet<>(),
-             new HashSet<>(),
-             new HashSet<>(),
-             new HashSet<>(),
-             new HashMap<>(),
-             new HashMap<>(),
-             new HashMap<>(),
+        this(new TreeSet<>(),
+             new TreeSet<>(),
+             new TreeSet<>(),
+             new TreeSet<>(),
+             new TreeSet<>(),
+             new TreeSet<>(),
+             new TreeMap<>(),
+             new TreeMap<>(),
+             new TreeMap<>(),
```

Comment:
Changed all these to treesets for readability when we print the states. I'm not sure if it's a performance concern. If so, we can back it out.